src/core/internal/convert.d

```diff
@@ -323,14 +366,42 @@ private ulong denormalizedMantissa(T)(T x) if(floatFormat!T == FloatFormat.Real8
 }
 
 @safe pure nothrow @nogc
-private ulong denormalizedMantissa(T)(T x) if(floatFormat!T != FloatFormat.Real80)
+private ulong denormalizedMantissa(T)(T x)
+    if(floatFormat!T == FloatFormat.Float || floatFormat!T == FloatFormat.Double)
 {
     x *= 2.0L^^FloatTraits!T.MANTISSA;
     auto fl = parse!true(x);
     ulong mant = fl.mantissa >> (FloatTraits!T.MANTISSA - fl.exponent);
     return shiftrRound(mant);
 }
 
+@safe pure nothrow @nogc
+private Float denormalizedMantissa(T)(T x, uint sign) if(floatFormat!T == FloatFormat.Quadruple)
```

Comment:
This doesn't overload the above functions and so should have a distinct name, ie: `floatDenormalizedMantissa`.

Having a look at all uses of the other function, they are all `Float(denormalizedMantissa(x), 0, sign)`, so they all could be refactored to return a Float as well as sharing the same signature.

Reply:
Good point, I'll fix it.